src/lib.rs

```diff
@@ -87,6 +87,11 @@ mod signal;
 #[cfg(feature = "signal")]
 pub use self::signal::{abort, raise, signal};
 
+#[cfg(feature = "memchr")]
```

Comment:
This doesn’t need to be gated - it’s fine to always include as the function is conditionally mangled.